packages/eslint-plugin/src/rules/no-magic-numbers.ts

```diff
@@ -55,6 +55,18 @@ export default util.createRule<Options, MessageIds>({
       return typeof node.value === 'number';
     }
 
+    /**
+     * Checks if the node grandparent is a Typescript enum declaration
+     * @param node the node to be validated.
+     * @returns true if the node grandparent is a Typescript enum declaration
+     * @private
+     */
+    function isGrandparentTSEnumDeclaration(node: TSESTree.Node): boolean {
+      return node.parent && node.parent.parent
+        ? node.parent.parent.type === AST_NODE_TYPES.TSEnumDeclaration
```

Comment:
no need for a ternary to switch between a boolean outcome

Reply:
I was trying to copy the style of all the other methods in that file. I'll change that.